src/i18n/locales/de-DE/accounts.json

```diff
@@ -0,0 +1,59 @@
+{
+    "show": {
+        "header": "Konto: {{account}}", // Account
+        "description": {
+            "title": "Konto Beschreibung:", // Account Description
+            "address": "Adresse:", // Address
+            "sentTransactions": "Sende Transaktionen:", // Sent Transactions
```

Comment:
"sentTransactions": "Gesendete Transaktionen: ", // Sent Transaktions

Reply:
I do not know how they mean it